demo/demo_notebooks/fraud_detection.ipynb

```diff
@@ -58,7 +58,7 @@
     "test_path            = \"/home/leonid/projects/database-stream-processor/demo/demo_notebooks/fraud_data/test.csv\"\n",
     "test_outpath         = \"/home/leonid/projects/database-stream-processor/demo/demo_notebooks/fraud_data/test_output.csv\"\n",
     "demographics_path    = \"/home/leonid/projects/database-stream-processor/demo/demo_notebooks/fraud_data/demographics.csv\"\n",
-    "simulation_path      = \"/home/leonid/projects/database-stream-processor/demo/demo_notebooks/fraud_data/simulation.csv\"\n",
+    "simulation_path      = \"/home/leonid/projects/database-stream-processor/demo/demo_notebooks/fraud_data/simulation_short.csv\"\n",
```

Comment:
are we supposed to review notebooks?
I'd like to, but this is not really readable

Reply:
It's ok to ignore them for now.